data_utils/serialized_dataset_loader.py

```diff
@@ -48,6 +48,8 @@ def load_serialized_data(
         """
         dataset = []
         with open(dataset_path, "rb") as f:
+            x_minmax = pickle.load(f)
```

Comment:
Do you use these values anywhere?

Reply:
Not used here, but not sure if we could skip the first two and load the third item directly

Reply:
Makes sense